openlibrary/core/lending.py

```diff
@@ -38,17 +38,25 @@
 
 IA_API_URL = "https://archive.org/services/openlibrary.php"
 AVAILABILITY_API = 'https://archive.org/services/loans/beta/loan/index.php'
+IA_USER_API_URL = "http://archive.org/~mek/auth.php"
```

Comment:
@LeadSongDog thanks for the review!
All URLs removed from this file to the config, all https (this auth.php service was for testing and has been replaced)